integration/spark/src/main/scala/org/apache/spark/sql/execution/command/index/DropIndexCommand.scala

```diff
@@ -184,10 +184,10 @@ private[sql] case class DropIndexCommand(
         parentCarbonTable = getRefreshedParentTable(sparkSession, dbName)
         val indexMetadata = parentCarbonTable.getIndexMetadata
         if (null != indexMetadata && null != indexMetadata.getIndexesMap) {
-          val hasCgFgIndexes =
-            !(indexMetadata.getIndexesMap.size() == 1 &&
-              indexMetadata.getIndexesMap.containsKey(IndexType.SI.getIndexProviderName))
-          if (hasCgFgIndexes) {
+          val hasCgFgIndexes = indexMetadata.getIndexesMap.size() != 0 &&
```

Comment:
Please add a comment, on which case, we need to set 'indexExists' to false

Reply:
added.